projects/igniteui-angular/src/lib/directives/input/input.directive.spec.ts

```diff
@@ -92,37 +92,36 @@ describe('IgxInput', () => {
         expect(igxInput.placeholder).toBe('Test');
     });
 
-    it('Should have an initial filled style when data bound.', async(() => {
+    it('Should have an initial filled style when data bound.', fakeAsync(() => {
         const fixture = TestBed.createComponent(InitiallyFilledInputComponent);
         fixture.detectChanges();
 
-        fixture.whenStable().then(() => {
-            fixture.detectChanges();
+        tick();
```

Comment:
We don't need to tick here. You could remove those two lines.

Reply:
This one should remain, because some of the styles checked need time to be applied.